### PR TITLE
NMS-13448: Fix doc character escaping

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [X] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13448

### CODE REVIEW

**Review by indigo423 — APPROVED** [review]
yes, the escaping is just `\{not-antora-replaced-attribute}`. Good catch!

File: docs/modules/operation/pages/service-assurance/monitors/HttpMonitor.adoc [hunk 1]
```diff
@@ -35,17 +35,17 @@ The response is parsed and a return code extracted and verified.
...
                            then {page-component-title} uses the node's label to set the `host` header value if
                            this parameter is set to true. Otherwise, {page-component-title} falls back using
-                           the node interface's IP address as `host` header value.                                   | false 
+                           the node interface's IP address as `host` header value.                                   | false
 | password kbd:[{}]
        | Specifies the password to use. Use this parameter with the user parameter to perform basic +
                            authentication. The user and password parameters are ignored when the basic-authentication parameter is defined.                | empty string
-| port              | A list of TCP ports to try connection on.                                     | 80,8080,8888 
-| retry                | Number of attempts to get a valid HTTP response/response-text.                             | 0 
+| port              | A list of TCP ports to try connection on.                                     | 80,8080,8888
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/HttpMonitor.adoc:47]:
```suggestion
| port              | A list of TCP ports to try to connect to.                                    | 80,8080,8888
```

File: docs/modules/operation/pages/ticketing/introduction.adoc [hunk 2]
```diff
@@ -14,5 +14,5 @@ To activate the ticketing integration, set the following properties in `$\{OPENN
                                                                        The `NullTicketerPlugin` does nothing when attempting to create/update/delete tickets.
 | `opennms.alarmTroubleTicketEnabled`    | `false`                   | Defines if the integration is enabled.
                                                                        If enabled, various links to control the issue state appear on the alarm details page.
-| opennms.alarmTroubleTicketLinkTemplate | `${\id}`                   | A template to generate a link to the issue, e.g., `\http://issues.opennms.org/browse/${\id}`
-|===
\ No newline at end of file
+| opennms.alarmTroubleTicketLinkTemplate | `$\{id}`                   | A template to generate a link to the issue, e.g., `\http://issues.opennms.org/browse/$\{id}`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/ticketing/introduction.adoc:17]:
```suggestion
| opennms.alarmTroubleTicketLinkTemplate | `$\{id}`                   | A template to generate a link to the issue; for example, `\http://issues.opennms.org/browse/$\{id}`
```

File: docs/modules/operation/pages/ticketing/remedy.adoc [hunk 3]
```diff
@@ -53,10 +53,10 @@ Now configure the plugin itself by setting the following properties in `$\{OPENN
 | `remedy.portname`                            | required | The Port name of the HPD web service
 | `remedy.createendpoint`                      | required | The endpoint location of the Create-HPD web service
 | `remedy.createportname`                      | required | The Port name of the Create-HPD web service
-| `remedy.targetgroups`                        | optional | Colon-separated list of Remedy groups to which created tickets may be assigned (`{\groupp}` below refers to values from this list)
-| `remedy.assignedgroup.{\groupp}`               | optional | Assigned group for the target group `{\groupp}`
-| `remedy.assignedsupportcompany.{\groupp}`      | optional | Assigned support company for the target group `{\groupp}`
-| `remedy.assignedsupportorganization.{\groupp}` | optional | Assigned support organization for the target group `{\groupp}`
+| `remedy.targetgroups`                        | optional | Colon-separated list of Remedy groups to which created tickets may be assigned (`\{group}` below refers to values from this list)
```
Comment (Bonrob2) [inline docs/modules/operation/pages/ticketing/remedy.adoc:56]:
```suggestion
| `remedy.targetgroups`                        | optional | Colon-separated list of Remedy groups to which created tickets may be assigned (`\{group}` below refers to values from this list).
```

File: docs/modules/reference/pages/configuration/tuning-activemq.adoc [hunk 4]
```diff
@@ -1,6 +1,6 @@
 = Tuning ActiveMQ
 
-The settings for the embedded _ActiveMQ_ broker are found in `$\{OPENNMS_HOME}/etc/opennms-activemq.xml`.
+The settings for the embedded _ActiveMQ_ broker are found in `$OPENNMS_HOME/etc/opennms-activemq.xml`.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-activemq.adoc:3]:
```suggestion
The settings for the embedded ActiveMQ broker are found in `$OPENNMS_HOME/etc/opennms-activemq.xml`.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]

File: docs/modules/development/pages/bsm/bsm-development.adoc [hunk 5]
```diff
@@ -145,6 +145,6 @@ The whole model is defined in `jetty-webapps/opennms/WEB-INF/lib/org.opennms.fea
 [options="header", cols="5,10"]
 |===
 | Resource                                                    | Description
-| `/opennms/api/v2/business-services/\{id\}`                  | Deletes the business service identified by `id`.
-| `/opennms/api/v2/business-services/\{id\}/edges/\{edgeid\}` | Removes an edge with the identifier `edgeId` from the business service identified by `id`.
+| `/opennms/api/v2/business-services/\{id}`                   | Deletes the business service identified by `id`.
+| `/opennms/api/v2/business-services/\{id}/edges/\{edgeid}`   | Removes an edge with the identifier `edgeId` from the business service identified by `id`.
```
[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/development/pages/bsm/bsm-development.adoc:149]: Yes. Thanks.

File: docs/modules/development/pages/rest/nodes.adoc [hunk 6]
```diff
@@ -1,7 +1,7 @@
 [[nodes-rest]]
 = Nodes
 
-NOTE: The default offset is 0, the default limit is 10 results.  
+NOTE: The default offset is 0, the default limit is 10 results.
```
[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/development/pages/rest/nodes.adoc:4]:
```suggestion
NOTE: The default offset is 0.
The default limit is 10 results.
```

**Review by Bonrob2 — APPROVED** [review]

**Pull request merged**